cengage/react-magma
# fix (Build) Updating emotion to v11

Issue: [#1335](https://github.com/cengage/react-magma/issues/1335)

## What I did

- Updated emotion to v11
- Updated unit test snapshots

## Screenshots:
 No visual changes expected

## Checklist 

## How to test


1.  install packages
2. ensure that the project runs locally and all parts build successfully
3. check all the components to see there are no new visual changes
